src/CodeCompliance/Domain/Checks/Filters/PluginFilter.php

```diff
@@ -17,6 +17,16 @@ class PluginFilter implements FilterInterface
      */
     public const PLUGIN_FILTER = 'PLUGIN_FILTER';
 
+    /**
+     * @var string
+     */
+    protected const PLUGIN_CLASS_SUFFIX = '/.*Plugin$/';
+
+    /**
+     * @var string
+     */
+    protected const PLUGIN_DIR_NAME = '/\/Plugin\//';
```

Comment:
I would prefer if we used # as separator, makes it easier to read and no need for extra escapes
```suggestion
    protected const PLUGIN_DIR_NAME = '#/Plugin/#';
```

Reply:
Yes, you a right. Deleted the constants. At least until we haven't another place for using them.